tests/Feature/CelebrityTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+
+class CelebrityTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_example()
+    {
+        $response = $this->get('/');
+
+        $response->assertStatus(200);
+    }
+
+
+    /** @test  */
+    public function test_users_can_browse_celebrity()
+    {
+        $user = User::factory()->create();
+
+        $response = $this->get('/celebrities');
+
+        $response -> assertStatus(200);
```

Comment:
So this can be simplified to something like this:

```
$user = User::factory()->create();

$this->actingAs($user)
       ->get('/celebrities')
       ->assertOk()
```
The example above is a bit more readable don't you think?
- So you create an user
- Acting as the user, you try to get to the `/celebrities` route, then the expected response should be an Ok (status code 200)